axelor-account/src/main/java/com/axelor/apps/account/service/move/MoveValidateServiceImpl.java

```diff
@@ -312,15 +314,19 @@ public void accounting(Move move, boolean updateCustomerAccount) throws AxelorEx
     moveInvoiceTermService.generateInvoiceTerms(move);
 
     this.completeMoveLines(move);
-
+    log.debug("After complete move lines");
     this.freezeAccountAndPartnerFieldsOnMoveLines(move);
+    log.debug("After freeze method");
 
     this.updateValidateStatus(move, dayBookMode);
 
+    log.debug("After status update");
     moveRepository.save(move);
 
+    log.debug("After save");
     if (updateCustomerAccount) {
       moveCustAccountService.updateCustomerAccount(move);
+      log.debug("After updateCustomerAccount");
```

Comment:
not sure that all these logs are necessary, this looks like they were used for debugging